### PR TITLE
Added glitch.py ☺️

### PR DESCRIPTION
**Pull request by Krishna-Singhal**

### CODE REVIEW

File: plugins/glitch.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+async def glitch_(message: Message):
+    """ Create Glitch effect in any media """
+    replied = message.reply_to_message
+    if not (replied and (
+            replied.photo or replied.sticker or replied.video or replied.animation)):
+        await message.err("```Media not found...```")
+        await message.reply_sticker('CAADBQADVAUAAjZgsCGE7PH3Wt1wSRYE')
+        return
+    if message.filtered_input_str:
+        input_ = int(message.filtered_input_str)
```
Comment (rking32) [inline plugins/glitch.py:31]: add this after Line34 unless its useless
```suggestion
```

**Comment by rking32** [comment]
@Krishna-Singhal fix this too
https://github.com/UsergeTeam/Userge-Plugins/blob/dc216a98fa9647890635eefecfd158b45865ad1c/plugins/transform.py#L152-L155

**Comment by rking32** [comment]
@Krishna-Singhal you are converting it to **integer** before checking

**Comment by rking32** [comment]
> @Krishna-Singhal fix this too
> 
> https://github.com/UsergeTeam/Userge-Plugins/blob/dc216a98fa9647890635eefecfd158b45865ad1c/plugins/transform.py#L152-L155

@Krishna-Singhal can you fix this one too

**Comment by Krishna-Singhal** [comment]
Now merge sir 😅😅

**Pull request merged**